src/develop/pixelpipe_hb.h

```diff
@@ -85,9 +85,9 @@ typedef enum dt_dev_pixelpipe_status_t
 typedef enum dt_dev_pixelpipe_stopper_t
 {
   DT_DEV_PIXELPIPE_STOP_NO = 0,
-  DT_DEV_PIXELPIPE_STOP_NODES = 1,
-  DT_DEV_PIXELPIPE_STOP_HQ = 2,
-  DT_DEV_PIXELPIPE_STOP_LAST = 2,
+  DT_DEV_PIXELPIPE_STOP_NODES,
+  DT_DEV_PIXELPIPE_STOP_HQ,
+  DT_DEV_PIXELPIPE_STOP_MODULE, // if more stoppers are added this must be the last
```

Comment:
Why did you rename? `_LAST` _is_ the usual name?

Reply:
To give a better hint to the intention, Anything >= STOP_MODULE indicating a module-intended shutdown? No good?

Reply:
Rethought, kept _LAST variant as it's common.